src/OrchardCore.Modules/OrchardCore.AdminMenu/Assets/scss/admin-menu-icon-picker.scss

```diff
@@ -1,54 +1,50 @@
-:root[data-theme='default'] {
-
-    @import "../../../../OrchardCore.Themes/TheAdmin/Assets/scss/modules/theme/default/_variables.scss";
-    @import "../../node_modules/bootstrap/scss/functions";
-    @import "../../node_modules/bootstrap/scss/variables";
+@import "../../../../OrchardCore.Themes/TheAdmin/Assets/scss/_variables";
+@import "../../node_modules/bootstrap/scss/_functions";
+@import "../../node_modules/bootstrap/scss/_variables";
+@import '../../node_modules/bootstrap/scss/mixins/_color-mode';
 
+@include color-mode(light) {
```

Comment:
This here needs that _color-mode mixin at least to work and probably _variables.

Reply:
yes it is needed and is imported. Do you see an issue?

Reply:
No issue, but you still need to import some Bootstrap mixins. At least not the entire Bootstrap CSS which I agree could be removed and used with a CDN instead now.